docs/sources/panels-visualizations/visualizations/xy-chart/index.md

```diff
@@ -0,0 +1,264 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/xy-chart/
+keywords:
+  - grafana
+  - chart
+  - xy chart
+  - documentation
+  - guide
+  - graph
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+description: Configure options for Grafana's xy chart
+title: XY chart
+weight: 100
+---
+
+# XY chart
+
+XY charts provide a way to visualize arbitrary x and y values in a graph so that you can easily show the relationship between two variables.
+
+![An xy chart showing height weight distribution](screenshot-xy-chart-v10.4.png)
+
+## Supported data formats
+
+You can use any type of tabular data in an xy chart. This type of visualization doesn't require time data.
```

Comment:
I'd slightly rephrase this. Two numeric fields are required for an XY chart so it can't support any tabular data (most of it though 😄).